frontend/src/libs/components/burger-menu/burger-menu.tsx

```diff
@@ -0,0 +1,94 @@
+import { AppRoute, Breakpoint, IconName } from '~/libs/enums/enums.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import {
+  useCallback,
+  useEffect,
+  useLocation,
+  useNavigate,
+  useState,
+} from '~/libs/hooks/hooks.js';
+import { type BurgerMenuItem } from '~/libs/types/types.js';
+
+import { Button, Icon } from '../components.js';
+import { BURGER_MENU_ITEMS } from './burger-menu-items.js';
+import styles from './styles.module.scss';
+
+let currentMenuItems: BurgerMenuItem[] | null;
```

Comment:
I don't like the idea of let variable outside the component, can u throw items through props?